tree/ntuple/v7/inc/ROOT/RPage.hxx

```diff
@@ -78,10 +78,10 @@ public:
 
    ColumnId_t GetColumnId() const { return fColumnId; }
    /// The space taken by column elements in the buffer
-   ClusterSize_t::ValueType GetNBytes() const { return fElementSize * fNElements; }
-   ClusterSize_t::ValueType GetElementSize() const { return fElementSize; }
-   ClusterSize_t::ValueType GetNElements() const { return fNElements; }
-   ClusterSize_t::ValueType GetMaxElements() const { return fMaxElements; }
+   std::uint32_t GetNBytes() const { return fElementSize * fNElements; }
```

Comment:
I think overflow is not an issue here, but maybe we could return `std::size_t` instead.
```suggestion
   std::size_t GetNBytes() const { return fElementSize * fNElements; }
```

Reply:
That's a good point.  I'd like to address this consistently in a separate PR though if needed.  The idea, currently, is to always use `std::uint32_t` for page sizes, implicitly saying "pages are small". It could also be `std::size_t`, saying (uncompressed) pages are objects in memory. I'm not sure actually which one to prefer.